### PR TITLE
Translate media widget

### PR DESCRIPTION
**Pull request by core23**
Fixed https://github.com/sonata-project/SonataMediaBundle/issues/888

Before
bildschirmfoto 2015-11-13 um 11 07 16

After

bildschirmfoto 2015-11-13 um 11 06 44

Translations
* [X] German
* [X] English
* [ ] French
* [X] Hungary

### CODE REVIEW

**Comment by core23** [comment]
@rande @Soullivaneuh French translation please :)

**Comment by OskarStark** [comment]
I would prefer "Löschen" @core23

**Comment by OskarStark** [comment]
ping @sagikazarmark

**Comment by sagikazarmark** [comment]
``` xml
<trans-unit id="widget_label_unlink">
    <source>widget_label_unlink</source>
    <target>Fájl törlése</target>
</trans-unit>
<trans-unit id="widget_label_binary_content">
    <source>widget_label_binary_content</source>
    <target>Fájl</target>
</trans-unit>
```

Sorry for not opening a PR.

**Comment by OskarStark** [comment]
@sagikazarmark awesome, thank you for your fast reply, this was ``hu`` or another language?

**Comment by core23** [comment]
Thank you @sagikazarmark

**Comment by sagikazarmark** [comment]
Sure, it is hu.

**Comment by OskarStark** [comment]
@core23 you added the translation from @sagikazarmark to ``fr`` instead of ``hu`` :smile:

**Comment by core23** [comment]
@OskarStark Damn, you're right. Thanks :sweat_smile:

File: Provider/BaseVideoProvider.php [hunk 1]
```diff
@@ -145,7 +145,9 @@ public function buildCreateForm(FormMapper $formMapper)
      */
     public function buildMediaType(FormBuilder $formBuilder)
     {
-        $formBuilder->add('binaryContent', 'text');
+        $formBuilder->add('binaryContent', 'text', array(
+            'label'    => 'widget_label_binary_content',
```
Comment (OskarStark) [inline Provider/BaseVideoProvider.php]: indention wrong for the arrow (``=>``)

Comment (core23) [inline Provider/BaseVideoProvider.php]: Fixed. Now we're waiting for the French guys ;) @rande @Soullivaneuh

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**